com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -59,7 +59,7 @@ public abstract partial class NetworkedBehaviour : MonoBehaviour
         /// <summary>
         /// Gets if we are executing as server
         /// </summary>
-        protected bool IsServer => IsRunning && NetworkingManager.Singleton.IsServer;
+        protected static bool IsServer => IsRunning && NetworkingManager.Singleton.IsServer;
```

Comment:
Why are we making this static?

Reply:
+1 — yeah, we probably shouldn't. probably a leftover? :)

Reply:
It's unfortunately deliberate...I need to know if I'm a server inside (static) `NetworkedBehaviourUpdate()`, before we did not.

Reply:
just use `NetworkingManager.Singleton` directly and do not modify here?
I mean, they're both the same thing under the hood but would make it easier to move away from statics and singletons easier if we had less places to deal with.

Reply:
If I try and use the NetworkingManager singletons directly, then I have to expand all 9 instances of `IsServer` into 

`NetworkingManager.Singleton != null && NetworkingManager.Singleton.IsListening && NetworkingManager.Singleton.IsServer`

which I would call out as a loss in clarity.  Also, another reason to make `IsServer` a static is simply to avoid allocating storage in every `NetworkedBehaviour` instance.